pkg/dao/repository_config.go

```diff
@@ -444,48 +447,62 @@ func (r repositoryConfigDaoImpl) ValidateName(orgId string, name string, respons
 	if name == "" {
 		response.Valid = false
 		response.Error = "Name cannot be blank"
-	} else {
-		found := models.RepositoryConfiguration{}
-		query := r.db.Where("name = ? AND ORG_ID = ?", name, orgId)
-		if len(excludedUUIDS) != 0 {
-			query = query.Where("repository_configurations.uuid NOT IN ?", excludedUUIDS)
-		}
-		result := query.Find(&found)
-		if result.Error != nil {
-			response.Valid = false
-			return result.Error
-		} else if found.UUID != "" {
-			response.Valid = false
-			response.Error = fmt.Sprintf("A repository with the name '%s' already exists.", name)
-		} else {
-			response.Valid = true
-		}
+		return nil
+	}
+
+	found := models.RepositoryConfiguration{}
+	query := r.db.Where("name = ? AND ORG_ID = ?", name, orgId)
+	if len(excludedUUIDS) != 0 {
+		query = query.Where("repository_configurations.uuid NOT IN ?", excludedUUIDS)
 	}
+	if err := query.Find(&found).Error; err != nil {
+		response.Valid = false
+		return err
+	}
+
+	if found.UUID != "" {
+		response.Valid = false
+		response.Error = fmt.Sprintf("A repository with the name '%s' already exists.", name)
+		return nil
+	}
+
+	response.Valid = true
 	return nil
 }
 
 func (r repositoryConfigDaoImpl) ValidateUrl(orgId string, url string, response *api.RepositoryValidationResponse, excludedUUIDS []string) error {
 	if url == "" {
 		response.URL.Valid = false
 		response.URL.Error = "URL cannot be blank"
-	} else {
-		found := models.RepositoryConfiguration{}
-		query := r.db.Preload("Repository").
-			Joins("inner join repositories on repository_configurations.repository_uuid = repositories.uuid").
-			Where("Repositories.URL = ? AND ORG_ID = ?", url, orgId)
-		if len(excludedUUIDS) != 0 {
-			query = query.Where("repository_configurations.uuid NOT IN ?", excludedUUIDS)
-		}
-		result := query.Find(&found)
-		if result.Error != nil {
-			response.URL.Valid = false
-			return result.Error
-		} else if found.UUID != "" {
-			response.URL.Valid = false
-			response.URL.Error = fmt.Sprintf("A repository with the URL '%s' already exists.", url)
-		} else {
-			response.URL.Valid = true
-		}
+		return nil
+	}
+
+	found := models.RepositoryConfiguration{}
+	query := r.db.Preload("Repository").
+		Joins("inner join repositories on repository_configurations.repository_uuid = repositories.uuid").
+		Where("Repositories.URL = ? AND ORG_ID = ?", url, orgId)
+	if len(excludedUUIDS) != 0 {
+		query = query.Where("repository_configurations.uuid NOT IN ?", excludedUUIDS)
+	}
+	if err := query.Find(&found).Error; err != nil {
+		response.URL.Valid = false
+		return err
+	}
+
+	if found.UUID != "" {
+		response.URL.Valid = false
+		response.URL.Error = fmt.Sprintf("A repository with the URL '%s' already exists.", url)
+		return nil
 	}
+
+	foundRepository := found.Repository
+	containsWhitespace := strings.ContainsAny(strings.TrimSpace(foundRepository.URL), " \t\n\v\r\f")
```

Comment:
```suggestion
	containsWhitespace := strings.ContainsAny(strings.TrimSpace(url), " \t\n\v\r\f")
```

Reply:
otherwise it pulls what was found from the db, which if nothing is found would be ""